### PR TITLE
Startup sequence

### PR DESCRIPTION
**Pull request by ConnorNovak**
- Corresponds to pull request on `state_controller` repository
- Adds startup sequence for properly calibrating tractor steering systems
- Fixes significant error: Estop pin for reading and writing were the same on the Teensy

### CODE REVIEW

**Comment by ConnorNovak** [comment]
Start reviewing, but don't merge yet - need to test on platform first

File: .gitignore [hunk 1]
```diff
@@ -51,3 +52,6 @@ scripts/CMakeFiles/
 # Ros Bag files and folder
 *.bag
 bags/
+
+# Local configurable changes
+src/hind_brain/lib
```
Comment (nathanestill) [inline .gitignore]: Remove that

File: src/HindBrain/HindBrain.h [hunk 2]
```diff
@@ -0,0 +1,90 @@
+/*
+  @file HindBrain.h
+  Purpose: Config file for HindBrain.ino
+*/
+
+#ifndef HIND_BRAIN_H
+#define HIND_BRAIN_H
+
+#define USE_USBCON
```
Comment (nathanestill) [inline src/HindBrain/HindBrain.h]: Add link

File: src/HindBrain/HindBrain.ino [hunk 3]
```diff
@@ -0,0 +1,374 @@
...
+  // Run startup & calibration sequence with appropriate user input.
+  //TODO create prompt publishing topic
+
+  nh.loginfo("Remove pins on steering and velocity, then publish 'y' to /user_input topic");
+  waitForUserVerification();
+
+  nh.loginfo("Verification received, homing steering and velocity actuators . . .");
+  delay(500);
+  rc1.SpeedAccelDeccelPositionM1(RC1_ADDRESS, 0, 300, 0, velMsg, 1);
+  rc1.SpeedAccelDeccelPositionM2(RC1_ADDRESS, 0, 500, 0, steerMsg, 1);
```
Comment (nathanestill) [inline src/HindBrain/HindBrain.ino]: I'm scared that these velMsg, steerMsg, and hitchMsg are declared but uninitialized at this point.

File: src/HindBrain/HindBrain.ino [hunk 4]
```diff
@@ -0,0 +1,374 @@
...
+      nh->logerror("Lost connectivity . . .");
+      eStopTractor();
+    }
+  }
+}
+
+
+char checkUserInput() {
+  // Get most recent user-sent input (if any), reset global var if necessary.
+  auto outMsg = usrMsg;
```
Comment (nathanestill) [inline src/HindBrain/HindBrain.ino]: again, should usr be initialized or is it ok?

**Review by nathanestill — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (ConnorNovak) [reply .gitignore]: resolved

File: src/HindBrain/HindBrain.h (hunk 2 above)
Comment (ConnorNovak) [reply src/HindBrain/HindBrain.h]: resolved

File: src/HindBrain/HindBrain.ino (hunk 4 above)
Comment (ConnorNovak) [reply src/HindBrain/HindBrain.ino]: Probably a race condition, fixed.

File: src/HindBrain/HindBrain.ino (hunk 3 above)
Comment (ConnorNovak) [reply src/HindBrain/HindBrain.ino]: Yup, a race condition. Fixed.

**Review by nathanestill — APPROVED** [review]

**Pull request merged**